test/tests/test_rest_framework_fields.py

```diff
@@ -1,12 +1,6 @@
-from __future__ import absolute_import, unicode_literals
-
 from rest_framework import serializers
 import sys
-
-if sys.version_info.major == 2:
-    import unittest2 as unittest
```

Comment:
Could you also drop `unittest2` from `tox.ini`?  This appears to be the only reference to it.

There are a couple of other references to Python 2 in `tox.ini` that you should drop along with this; search for `py27`.

Reply:
done :-)